shared/video-sdk/develop/cloud-proxy/project-setup/electron.mdx

```diff
@@ -0,0 +1,3 @@
+<PlatformWrapper platform="electron">
+
```

Comment:
In docs/shared/video-sdk/develop/_cloud-proxy.mdx (not changed as a part of this PR but still can be fixed in this) `[<Vpl k="CONSOLE" />]` is not rendering and needs to be changed to `[<Vg k="CONSOLE" />]`